From c5c084f6a4a01ed5e6d2f90cf145a289c9d2ee94 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 20 Aug 2012 10:00:57 +1000
Subject: [PATCH] Don't force pInfo->flags to 0 (#40870)

The server uses the flags to decide if a device comes up as attached or
floating slave. Overwriting the flag after the server has processed the
CorePointer/AlwaysCore/Floating/etc options results in the device always
coming up as floating slave. This again makes the device appear
unresponsive.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
(cherry picked from commit 00742d9a9bcd06800b2548007ea304c331875666)

Conflicts:
	src/xf86Fpit.c
---
 src/xf86Fpit.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/xf86Fpit.c b/src/xf86Fpit.c
index a156745..87ca93c 100644
--- a/src/xf86Fpit.c
+++ b/src/xf86Fpit.c
@@ -528,7 +528,6 @@ static int xf86FpitAllocate(InputDriverPtr drv, InputInfoPtr pInfo)
 	priv->fpitIndex = 0;
 	priv->fpitPassive = 0;
 	pInfo->name = strdup(XI_TOUCHSCREEN);
-	pInfo->flags = 0 /* XI86_NO_OPEN_ON_INIT */ ;
 	pInfo->device_control = xf86FpitControl;
 	pInfo->read_input = xf86FpitReadInput;
 	pInfo->control_proc = NULL;
-- 
1.7.11.2

